!SESSION 2016-08-05 10:35:54.289 -----------------------------------------------
eclipse.buildId=4.6.0.I20160606-1100
java.version=1.8.0_101
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=zh_CN
Framework arguments:  -product org.eclipse.epp.package.jee.product
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.jee.product

!ENTRY org.eclipse.egit.ui 2 0 2016-08-05 10:36:37.123
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\Rorsc'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY org.python.pydev.shared_core 1 1 2016-08-05 10:37:31.540
!MESSAGE Information about process of adding new interpreter:
- Opening dialog to request executable (or jar).
- Chosen interpreter (name and file):'Tuple [Python34 -- C:\Python34\python.exe]
- Ok, file is non-null. Getting info on:C:\Python34\python.exe
- Beggining task:Getting libs totalWork:100
- Setting task name:Mounting executable string...
- Setting task name:Executing:  C:\Python34\python.exe -u D:\DEV\eclipse-jee-neon-R-win32-x86_64\eclipse\plugins\org.python.pydev_5.1.2.201606231256\pysrc\interpreterInfo.py
- Setting task name:Making pythonpath environment... C:\Python34\python.exe -u D:\DEV\eclipse-jee-neon-R-win32-x86_64\eclipse\plugins\org.python.pydev_5.1.2.201606231256\pysrc\interpreterInfo.py
- Setting task name:Making exec... C:\Python34\python.exe -u D:\DEV\eclipse-jee-neon-R-win32-x86_64\eclipse\plugins\org.python.pydev_5.1.2.201606231256\pysrc\interpreterInfo.py
- Setting task name:Reading output...
- Setting task name:Waiting for process to finish.
- Success getting the info. Result:<xml>
<name>Python34</name>
<version>3.4</version>
<executable>C:\Python34\python.exe</executable>
<lib>C:\Python34\DLLs</lib>
<lib>C:\Python34\lib</lib>
<lib>C:\Python34</lib>
<lib>C:\Python34\lib\site-packages</lib>
<forced_lib>Image</forced_lib>
<forced_lib>OpenGL</forced_lib>
<forced_lib>_ast</forced_lib>
<forced_lib>_bisect</forced_lib>
<forced_lib>_bytesio</forced_lib>
<forced_lib>_codecs</forced_lib>
<forced_lib>_codecs_cn</forced_lib>
<forced_lib>_codecs_hk</forced_lib>
<forced_lib>_codecs_iso2022</forced_lib>
<forced_lib>_codecs_jp</forced_lib>
<forced_lib>_codecs_kr</forced_lib>
<forced_lib>_codecs_tw</forced_lib>
<forced_lib>_collections</forced_lib>
<forced_lib>_csv</forced_lib>
<forced_lib>_datetime</forced_lib>
<forced_lib>_fileio</forced_lib>
<forced_lib>_functools</forced_lib>
<forced_lib>_heapq</forced_lib>
<forced_lib>_hotshot</forced_lib>
<forced_lib>_imp</forced_lib>
<forced_lib>_io</forced_lib>
<forced_lib>_json</forced_lib>
<forced_lib>_locale</forced_lib>
<forced_lib>_lsprof</forced_lib>
<forced_lib>_md5</forced_lib>
<forced_lib>_multibytecodec</forced_lib>
<forced_lib>_opcode</forced_lib>
<forced_lib>_operator</forced_lib>
<forced_lib>_pickle</forced_lib>
<forced_lib>_random</forced_lib>
<forced_lib>_sha</forced_lib>
<forced_lib>_sha1</forced_lib>
<forced_lib>_sha256</forced_lib>
<forced_lib>_sha512</forced_lib>
<forced_lib>_sre</forced_lib>
<forced_lib>_stat</forced_lib>
<forced_lib>_string</forced_lib>
<forced_lib>_struct</forced_lib>
<forced_lib>_subprocess</forced_lib>
<forced_lib>_symtable</forced_lib>
<forced_lib>_thread</forced_lib>
<forced_lib>_tracemalloc</forced_lib>
<forced_lib>_warnings</forced_lib>
<forced_lib>_weakref</forced_lib>
<forced_lib>_winapi</forced_lib>
<forced_lib>_winreg</forced_lib>
<forced_lib>array</forced_lib>
<forced_lib>atexit</forced_lib>
<forced_lib>audioop</forced_lib>
<forced_lib>binascii</forced_lib>
<forced_lib>builtins</forced_lib>
<forced_lib>cPickle</forced_lib>
<forced_lib>cStringIO</forced_lib>
<forced_lib>cmath</forced_lib>
<forced_lib>datetime</forced_lib>
<forced_lib>email</forced_lib>
<forced_lib>errno</forced_lib>
<forced_lib>exceptions</forced_lib>
<forced_lib>faulthandler</forced_lib>
<forced_lib>future_builtins</forced_lib>
<forced_lib>gc</forced_lib>
<forced_lib>gi</forced_lib>
<forced_lib>hashlib</forced_lib>
<forced_lib>imageop</forced_lib>
<forced_lib>imp</forced_lib>
<forced_lib>itertools</forced_lib>
<forced_lib>marshal</forced_lib>
<forced_lib>math</forced_lib>
<forced_lib>mmap</forced_lib>
<forced_lib>mock</forced_lib>
<forced_lib>msvcrt</forced_lib>
<forced_lib>multiprocessing</forced_lib>
<forced_lib>nt</forced_lib>
<forced_lib>numpy</forced_lib>
<forced_lib>operator</forced_lib>
<forced_lib>os</forced_lib>
<forced_lib>os.path</forced_lib>
<forced_lib>parser</forced_lib>
<forced_lib>pytest</forced_lib>
<forced_lib>scipy</forced_lib>
<forced_lib>signal</forced_lib>
<forced_lib>socket</forced_lib>
<forced_lib>strop</forced_lib>
<forced_lib>sys</forced_lib>
<forced_lib>thread</forced_lib>
<forced_lib>time</forced_lib>
<forced_lib>winreg</forced_lib>
<forced_lib>wx</forced_lib>
<forced_lib>wxPython</forced_lib>
<forced_lib>xxsubtype</forced_lib>
<forced_lib>zipimport</forced_lib>
<forced_lib>zlib</forced_lib>
</xml>

!STACK 0
java.lang.RuntimeException: Information about process of adding new interpreter:
- Opening dialog to request executable (or jar).
- Chosen interpreter (name and file):'Tuple [Python34 -- C:\Python34\python.exe]
- Ok, file is non-null. Getting info on:C:\Python34\python.exe
- Beggining task:Getting libs totalWork:100
- Setting task name:Mounting executable string...
- Setting task name:Executing:  C:\Python34\python.exe -u D:\DEV\eclipse-jee-neon-R-win32-x86_64\eclipse\plugins\org.python.pydev_5.1.2.201606231256\pysrc\interpreterInfo.py
- Setting task name:Making pythonpath environment... C:\Python34\python.exe -u D:\DEV\eclipse-jee-neon-R-win32-x86_64\eclipse\plugins\org.python.pydev_5.1.2.201606231256\pysrc\interpreterInfo.py
- Setting task name:Making exec... C:\Python34\python.exe -u D:\DEV\eclipse-jee-neon-R-win32-x86_64\eclipse\plugins\org.python.pydev_5.1.2.201606231256\pysrc\interpreterInfo.py
- Setting task name:Reading output...
- Setting task name:Waiting for process to finish.
- Success getting the info. Result:<xml>
<name>Python34</name>
<version>3.4</version>
<executable>C:\Python34\python.exe</executable>
<lib>C:\Python34\DLLs</lib>
<lib>C:\Python34\lib</lib>
<lib>C:\Python34</lib>
<lib>C:\Python34\lib\site-packages</lib>
<forced_lib>Image</forced_lib>
<forced_lib>OpenGL</forced_lib>
<forced_lib>_ast</forced_lib>
<forced_lib>_bisect</forced_lib>
<forced_lib>_bytesio</forced_lib>
<forced_lib>_codecs</forced_lib>
<forced_lib>_codecs_cn</forced_lib>
<forced_lib>_codecs_hk</forced_lib>
<forced_lib>_codecs_iso2022</forced_lib>
<forced_lib>_codecs_jp</forced_lib>
<forced_lib>_codecs_kr</forced_lib>
<forced_lib>_codecs_tw</forced_lib>
<forced_lib>_collections</forced_lib>
<forced_lib>_csv</forced_lib>
<forced_lib>_datetime</forced_lib>
<forced_lib>_fileio</forced_lib>
<forced_lib>_functools</forced_lib>
<forced_lib>_heapq</forced_lib>
<forced_lib>_hotshot</forced_lib>
<forced_lib>_imp</forced_lib>
<forced_lib>_io</forced_lib>
<forced_lib>_json</forced_lib>
<forced_lib>_locale</forced_lib>
<forced_lib>_lsprof</forced_lib>
<forced_lib>_md5</forced_lib>
<forced_lib>_multibytecodec</forced_lib>
<forced_lib>_opcode</forced_lib>
<forced_lib>_operator</forced_lib>
<forced_lib>_pickle</forced_lib>
<forced_lib>_random</forced_lib>
<forced_lib>_sha</forced_lib>
<forced_lib>_sha1</forced_lib>
<forced_lib>_sha256</forced_lib>
<forced_lib>_sha512</forced_lib>
<forced_lib>_sre</forced_lib>
<forced_lib>_stat</forced_lib>
<forced_lib>_string</forced_lib>
<forced_lib>_struct</forced_lib>
<forced_lib>_subprocess</forced_lib>
<forced_lib>_symtable</forced_lib>
<forced_lib>_thread</forced_lib>
<forced_lib>_tracemalloc</forced_lib>
<forced_lib>_warnings</forced_lib>
<forced_lib>_weakref</forced_lib>
<forced_lib>_winapi</forced_lib>
<forced_lib>_winreg</forced_lib>
<forced_lib>array</forced_lib>
<forced_lib>atexit</forced_lib>
<forced_lib>audioop</forced_lib>
<forced_lib>binascii</forced_lib>
<forced_lib>builtins</forced_lib>
<forced_lib>cPickle</forced_lib>
<forced_lib>cStringIO</forced_lib>
<forced_lib>cmath</forced_lib>
<forced_lib>datetime</forced_lib>
<forced_lib>email</forced_lib>
<forced_lib>errno</forced_lib>
<forced_lib>exceptions</forced_lib>
<forced_lib>faulthandler</forced_lib>
<forced_lib>future_builtins</forced_lib>
<forced_lib>gc</forced_lib>
<forced_lib>gi</forced_lib>
<forced_lib>hashlib</forced_lib>
<forced_lib>imageop</forced_lib>
<forced_lib>imp</forced_lib>
<forced_lib>itertools</forced_lib>
<forced_lib>marshal</forced_lib>
<forced_lib>math</forced_lib>
<forced_lib>mmap</forced_lib>
<forced_lib>mock</forced_lib>
<forced_lib>msvcrt</forced_lib>
<forced_lib>multiprocessing</forced_lib>
<forced_lib>nt</forced_lib>
<forced_lib>numpy</forced_lib>
<forced_lib>operator</forced_lib>
<forced_lib>os</forced_lib>
<forced_lib>os.path</forced_lib>
<forced_lib>parser</forced_lib>
<forced_lib>pytest</forced_lib>
<forced_lib>scipy</forced_lib>
<forced_lib>signal</forced_lib>
<forced_lib>socket</forced_lib>
<forced_lib>strop</forced_lib>
<forced_lib>sys</forced_lib>
<forced_lib>thread</forced_lib>
<forced_lib>time</forced_lib>
<forced_lib>winreg</forced_lib>
<forced_lib>wx</forced_lib>
<forced_lib>wxPython</forced_lib>
<forced_lib>xxsubtype</forced_lib>
<forced_lib>zipimport</forced_lib>
<forced_lib>zlib</forced_lib>
</xml>

	at org.python.pydev.core.log.Log.logInfo(Log.java:66)
	at org.python.pydev.ui.pythonpathconf.AbstractInterpreterEditor.getNewInputObject(AbstractInterpreterEditor.java:990)
	at org.python.copiedfromeclipsesrc.PythonListEditor.addPressed(PythonListEditor.java:132)
	at org.python.copiedfromeclipsesrc.PythonListEditor$1.widgetSelected(PythonListEditor.java:225)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:249)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:84)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4410)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1079)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:4228)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3816)
	at org.eclipse.jface.window.Window.runEventLoop(Window.java:818)
	at org.eclipse.jface.window.Window.open(Window.java:794)
	at org.eclipse.ui.internal.dialogs.WorkbenchPreferenceDialog.open(WorkbenchPreferenceDialog.java:211)
	at org.eclipse.ui.internal.OpenPreferencesAction.run(OpenPreferencesAction.java:63)
	at org.eclipse.jface.action.Action.runWithEvent(Action.java:473)
	at org.eclipse.jface.action.ActionContributionItem.handleWidgetSelection(ActionContributionItem.java:565)
	at org.eclipse.jface.action.ActionContributionItem.lambda$4(ActionContributionItem.java:397)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:84)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4410)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1079)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:4228)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3816)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$4.run(PartRenderingEngine.java:1121)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:336)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1022)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:150)
	at org.eclipse.ui.internal.Workbench$5.run(Workbench.java:687)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:336)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:604)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:148)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:138)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:673)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:610)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1519)

!ENTRY org.python.pydev.shared_core 1 1 2016-08-05 10:37:33.615
!MESSAGE Restoring info for: C:\Python34\python.exe
!STACK 0
java.lang.RuntimeException: Restoring info for: C:\Python34\python.exe
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.getInterpretersFromPersistedString(AbstractInterpreterManager.java:529)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.internalRecreateCacheGetInterpreterInfos(AbstractInterpreterManager.java:345)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.setInfos(AbstractInterpreterManager.java:655)
	at org.python.pydev.ui.pythonpathconf.AbstractInterpreterPreferencesPage$5.run(AbstractInterpreterPreferencesPage.java:282)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:119)
Caused by: java.io.IOException: Expecting: D:\DEV\request\.metadata\.plugins\org.python.pydev\v1_373nx5nhse1xrnikxmh9dta8v\modulesKeys to exist (and be a file).
	at org.python.pydev.editor.codecompletion.revisited.ModulesManager.loadFromFile(ModulesManager.java:293)
	at org.python.pydev.editor.codecompletion.revisited.SystemModulesManager.load(SystemModulesManager.java:386)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.getInterpretersFromPersistedString(AbstractInterpreterManager.java:527)
	... 4 more

!ENTRY org.python.pydev.shared_core 1 1 2016-08-05 10:37:34.295
!MESSAGE Finished restoring information for: C:\Python34\python.exe at: D:\DEV\request\.metadata\.plugins\org.python.pydev\v1_373nx5nhse1xrnikxmh9dta8v
!STACK 0
java.lang.RuntimeException: Finished restoring information for: C:\Python34\python.exe at: D:\DEV\request\.metadata\.plugins\org.python.pydev\v1_373nx5nhse1xrnikxmh9dta8v
	at org.python.pydev.core.log.Log.logInfo(Log.java:66)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.getInterpretersFromPersistedString(AbstractInterpreterManager.java:574)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.internalRecreateCacheGetInterpreterInfos(AbstractInterpreterManager.java:345)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.setInfos(AbstractInterpreterManager.java:655)
	at org.python.pydev.ui.pythonpathconf.AbstractInterpreterPreferencesPage$5.run(AbstractInterpreterPreferencesPage.java:282)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:119)

!ENTRY org.python.pydev.shared_core 4 4 2016-08-05 10:37:34.848
!MESSAGE Info: Rebuilding internal caches: D:\DEV\request\.metadata\.plugins\com.python.pydev.analysis\python_v1_373nx5nhse1xrnikxmh9dta8v\python.pydevsysteminfo (Expected error to be provided and got no error!)

!ENTRY org.python.pydev.shared_core 4 4 2016-08-05 10:37:40.492
!MESSAGE Did not expect to find item below node: Assign[targets=[Attribute[value=Name[id=self, ctx=Load, reserved=false], attr=NameTok[id=hard_break, ctx=Attrib], ctx=Load], Attribute[value=Name[id=self, ctx=Load, reserved=false], attr=NameTok[id=nospace, ctx=Attrib], ctx=Load]], value=null] (module: formatter.py).
!STACK 0
java.lang.RuntimeException: Did not expect to find item below node: Assign[targets=[Attribute[value=Name[id=self, ctx=Load, reserved=false], attr=NameTok[id=hard_break, ctx=Attrib], ctx=Load], Attribute[value=Name[id=self, ctx=Load, reserved=false], attr=NameTok[id=nospace, ctx=Attrib], ctx=Load]], value=null] (module: formatter.py).
	at org.python.pydev.core.log.Log.log(Log.java:54)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.endScopesInStack(FastDefinitionsParser.java:589)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.handleNewLine(FastDefinitionsParser.java:380)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.extractBody(FastDefinitionsParser.java:249)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.parse(FastDefinitionsParser.java:676)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalTokensInfo.addAstInfo(AbstractAdditionalTokensInfo.java:314)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalInfoWithBuild.restoreInfoForModuleManager(AbstractAdditionalInfoWithBuild.java:264)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.recreateAllInfo(AdditionalSystemInterpreterInfo.java:179)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.getAdditionalSystemInfo(AdditionalSystemInterpreterInfo.java:155)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.getAdditionalSystemInfo(AdditionalSystemInterpreterInfo.java:124)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.recreateAllInfo(AdditionalSystemInterpreterInfo.java:171)
	at com.python.pydev.analysis.additionalinfo.builders.InterpreterObserver.notifyDefaultPythonpathRestored(InterpreterObserver.java:36)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.restorePythopathForInterpreters(AbstractInterpreterManager.java:740)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.setInfos(AbstractInterpreterManager.java:657)
	at org.python.pydev.ui.pythonpathconf.AbstractInterpreterPreferencesPage$5.run(AbstractInterpreterPreferencesPage.java:282)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:119)

!ENTRY org.python.pydev.shared_core 4 4 2016-08-05 10:37:40.498
!MESSAGE Did not expect to find item below node: Assign[targets=[Attribute[value=Name[id=self, ctx=Load, reserved=false], attr=NameTok[id=nospace, ctx=Attrib], ctx=Load], Attribute[value=Name[id=self, ctx=Load, reserved=false], attr=NameTok[id=para_end, ctx=Attrib], ctx=Load]], value=null] (module: formatter.py).
!STACK 0
java.lang.RuntimeException: Did not expect to find item below node: Assign[targets=[Attribute[value=Name[id=self, ctx=Load, reserved=false], attr=NameTok[id=nospace, ctx=Attrib], ctx=Load], Attribute[value=Name[id=self, ctx=Load, reserved=false], attr=NameTok[id=para_end, ctx=Attrib], ctx=Load]], value=null] (module: formatter.py).
	at org.python.pydev.core.log.Log.log(Log.java:54)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.endScopesInStack(FastDefinitionsParser.java:589)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.handleNewLine(FastDefinitionsParser.java:380)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.extractBody(FastDefinitionsParser.java:249)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.parse(FastDefinitionsParser.java:676)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalTokensInfo.addAstInfo(AbstractAdditionalTokensInfo.java:314)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalInfoWithBuild.restoreInfoForModuleManager(AbstractAdditionalInfoWithBuild.java:264)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.recreateAllInfo(AdditionalSystemInterpreterInfo.java:179)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.getAdditionalSystemInfo(AdditionalSystemInterpreterInfo.java:155)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.getAdditionalSystemInfo(AdditionalSystemInterpreterInfo.java:124)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.recreateAllInfo(AdditionalSystemInterpreterInfo.java:171)
	at com.python.pydev.analysis.additionalinfo.builders.InterpreterObserver.notifyDefaultPythonpathRestored(InterpreterObserver.java:36)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.restorePythopathForInterpreters(AbstractInterpreterManager.java:740)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.setInfos(AbstractInterpreterManager.java:657)
	at org.python.pydev.ui.pythonpathconf.AbstractInterpreterPreferencesPage$5.run(AbstractInterpreterPreferencesPage.java:282)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:119)

!ENTRY org.python.pydev.shared_core 4 4 2016-08-05 10:37:40.503
!MESSAGE Did not expect to find item below node: Assign[targets=[Attribute[value=Name[id=self, ctx=Load, reserved=false], attr=NameTok[id=hard_break, ctx=Attrib], ctx=Load], Attribute[value=Name[id=self, ctx=Load, reserved=false], attr=NameTok[id=para_end, ctx=Attrib], ctx=Load]], value=null] (module: formatter.py).
!STACK 0
java.lang.RuntimeException: Did not expect to find item below node: Assign[targets=[Attribute[value=Name[id=self, ctx=Load, reserved=false], attr=NameTok[id=hard_break, ctx=Attrib], ctx=Load], Attribute[value=Name[id=self, ctx=Load, reserved=false], attr=NameTok[id=para_end, ctx=Attrib], ctx=Load]], value=null] (module: formatter.py).
	at org.python.pydev.core.log.Log.log(Log.java:54)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.endScopesInStack(FastDefinitionsParser.java:589)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.handleNewLine(FastDefinitionsParser.java:380)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.extractBody(FastDefinitionsParser.java:249)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.parse(FastDefinitionsParser.java:676)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalTokensInfo.addAstInfo(AbstractAdditionalTokensInfo.java:314)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalInfoWithBuild.restoreInfoForModuleManager(AbstractAdditionalInfoWithBuild.java:264)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.recreateAllInfo(AdditionalSystemInterpreterInfo.java:179)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.getAdditionalSystemInfo(AdditionalSystemInterpreterInfo.java:155)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.getAdditionalSystemInfo(AdditionalSystemInterpreterInfo.java:124)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.recreateAllInfo(AdditionalSystemInterpreterInfo.java:171)
	at com.python.pydev.analysis.additionalinfo.builders.InterpreterObserver.notifyDefaultPythonpathRestored(InterpreterObserver.java:36)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.restorePythopathForInterpreters(AbstractInterpreterManager.java:740)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.setInfos(AbstractInterpreterManager.java:657)
	at org.python.pydev.ui.pythonpathconf.AbstractInterpreterPreferencesPage$5.run(AbstractInterpreterPreferencesPage.java:282)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:119)

!ENTRY org.python.pydev.shared_core 4 4 2016-08-05 10:37:42.698
!MESSAGE Did not expect to find item below node: Assign[targets=[Name[id=module_name, ctx=Store, reserved=false]], value=null] (module: module.py).
!STACK 0
java.lang.RuntimeException: Did not expect to find item below node: Assign[targets=[Name[id=module_name, ctx=Store, reserved=false]], value=null] (module: module.py).
	at org.python.pydev.core.log.Log.log(Log.java:54)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.endScopesInStack(FastDefinitionsParser.java:589)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.handleNewLine(FastDefinitionsParser.java:380)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.extractBody(FastDefinitionsParser.java:249)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.parse(FastDefinitionsParser.java:676)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalTokensInfo.addAstInfo(AbstractAdditionalTokensInfo.java:314)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalInfoWithBuild.restoreInfoForModuleManager(AbstractAdditionalInfoWithBuild.java:264)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.recreateAllInfo(AdditionalSystemInterpreterInfo.java:179)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.getAdditionalSystemInfo(AdditionalSystemInterpreterInfo.java:155)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.getAdditionalSystemInfo(AdditionalSystemInterpreterInfo.java:124)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.recreateAllInfo(AdditionalSystemInterpreterInfo.java:171)
	at com.python.pydev.analysis.additionalinfo.builders.InterpreterObserver.notifyDefaultPythonpathRestored(InterpreterObserver.java:36)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.restorePythopathForInterpreters(AbstractInterpreterManager.java:740)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.setInfos(AbstractInterpreterManager.java:657)
	at org.python.pydev.ui.pythonpathconf.AbstractInterpreterPreferencesPage$5.run(AbstractInterpreterPreferencesPage.java:282)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:119)

!ENTRY org.python.pydev.shared_core 4 4 2016-08-05 10:37:42.980
!MESSAGE Did not expect to find item below node: Assign[targets=[Attribute[value=Name[id=self, ctx=Load, reserved=false], attr=NameTok[id=_weakref, ctx=Attrib], ctx=Load], Attribute[value=Name[id=self, ctx=Load, reserved=false], attr=NameTok[id=_callback, ctx=Attrib], ctx=Load]], value=null] (module: util.py).
!STACK 0
java.lang.RuntimeException: Did not expect to find item below node: Assign[targets=[Attribute[value=Name[id=self, ctx=Load, reserved=false], attr=NameTok[id=_weakref, ctx=Attrib], ctx=Load], Attribute[value=Name[id=self, ctx=Load, reserved=false], attr=NameTok[id=_callback, ctx=Attrib], ctx=Load]], value=null] (module: util.py).
	at org.python.pydev.core.log.Log.log(Log.java:54)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.endScopesInStack(FastDefinitionsParser.java:589)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.handleNewLine(FastDefinitionsParser.java:380)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.extractBody(FastDefinitionsParser.java:249)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.parse(FastDefinitionsParser.java:676)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalTokensInfo.addAstInfo(AbstractAdditionalTokensInfo.java:314)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalInfoWithBuild.restoreInfoForModuleManager(AbstractAdditionalInfoWithBuild.java:264)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.recreateAllInfo(AdditionalSystemInterpreterInfo.java:179)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.getAdditionalSystemInfo(AdditionalSystemInterpreterInfo.java:155)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.getAdditionalSystemInfo(AdditionalSystemInterpreterInfo.java:124)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.recreateAllInfo(AdditionalSystemInterpreterInfo.java:171)
	at com.python.pydev.analysis.additionalinfo.builders.InterpreterObserver.notifyDefaultPythonpathRestored(InterpreterObserver.java:36)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.restorePythopathForInterpreters(AbstractInterpreterManager.java:740)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.setInfos(AbstractInterpreterManager.java:657)
	at org.python.pydev.ui.pythonpathconf.AbstractInterpreterPreferencesPage$5.run(AbstractInterpreterPreferencesPage.java:282)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:119)

!ENTRY org.python.pydev.shared_core 4 4 2016-08-05 10:37:46.246
!MESSAGE Did not expect to find item below node: Assign[targets=[Name[id=oid, ctx=Store, reserved=false]], value=null] (module: encoder.py).
!STACK 0
java.lang.RuntimeException: Did not expect to find item below node: Assign[targets=[Name[id=oid, ctx=Store, reserved=false]], value=null] (module: encoder.py).
	at org.python.pydev.core.log.Log.log(Log.java:54)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.endScopesInStack(FastDefinitionsParser.java:589)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.handleNewLine(FastDefinitionsParser.java:380)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.extractBody(FastDefinitionsParser.java:249)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.parse(FastDefinitionsParser.java:676)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalTokensInfo.addAstInfo(AbstractAdditionalTokensInfo.java:314)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalInfoWithBuild.restoreInfoForModuleManager(AbstractAdditionalInfoWithBuild.java:264)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.recreateAllInfo(AdditionalSystemInterpreterInfo.java:179)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.getAdditionalSystemInfo(AdditionalSystemInterpreterInfo.java:155)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.getAdditionalSystemInfo(AdditionalSystemInterpreterInfo.java:124)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.recreateAllInfo(AdditionalSystemInterpreterInfo.java:171)
	at com.python.pydev.analysis.additionalinfo.builders.InterpreterObserver.notifyDefaultPythonpathRestored(InterpreterObserver.java:36)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.restorePythopathForInterpreters(AbstractInterpreterManager.java:740)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.setInfos(AbstractInterpreterManager.java:657)
	at org.python.pydev.ui.pythonpathconf.AbstractInterpreterPreferencesPage$5.run(AbstractInterpreterPreferencesPage.java:282)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:119)

!ENTRY org.python.pydev.shared_core 4 4 2016-08-05 10:37:46.298
!MESSAGE Did not expect to find item below node: Assign[targets=[Name[id=defaultValue, ctx=Store, reserved=false]], value=null] (module: base.py).
!STACK 0
java.lang.RuntimeException: Did not expect to find item below node: Assign[targets=[Name[id=defaultValue, ctx=Store, reserved=false]], value=null] (module: base.py).
	at org.python.pydev.core.log.Log.log(Log.java:54)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.endScopesInStack(FastDefinitionsParser.java:589)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.handleNewLine(FastDefinitionsParser.java:380)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.extractBody(FastDefinitionsParser.java:249)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.parse(FastDefinitionsParser.java:676)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalTokensInfo.addAstInfo(AbstractAdditionalTokensInfo.java:314)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalInfoWithBuild.restoreInfoForModuleManager(AbstractAdditionalInfoWithBuild.java:264)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.recreateAllInfo(AdditionalSystemInterpreterInfo.java:179)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.getAdditionalSystemInfo(AdditionalSystemInterpreterInfo.java:155)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.getAdditionalSystemInfo(AdditionalSystemInterpreterInfo.java:124)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.recreateAllInfo(AdditionalSystemInterpreterInfo.java:171)
	at com.python.pydev.analysis.additionalinfo.builders.InterpreterObserver.notifyDefaultPythonpathRestored(InterpreterObserver.java:36)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.restorePythopathForInterpreters(AbstractInterpreterManager.java:740)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.setInfos(AbstractInterpreterManager.java:657)
	at org.python.pydev.ui.pythonpathconf.AbstractInterpreterPreferencesPage$5.run(AbstractInterpreterPreferencesPage.java:282)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:119)

!ENTRY org.python.pydev.shared_core 4 4 2016-08-05 10:37:46.388
!MESSAGE Did not expect to find item below node: Assign[targets=[Name[id=r, ctx=Store, reserved=false]], value=null] (module: univ.py).
!STACK 0
java.lang.RuntimeException: Did not expect to find item below node: Assign[targets=[Name[id=r, ctx=Store, reserved=false]], value=null] (module: univ.py).
	at org.python.pydev.core.log.Log.log(Log.java:54)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.endScopesInStack(FastDefinitionsParser.java:589)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.handleNewLine(FastDefinitionsParser.java:380)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.extractBody(FastDefinitionsParser.java:249)
	at org.python.pydev.parser.fastparser.FastDefinitionsParser.parse(FastDefinitionsParser.java:676)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalTokensInfo.addAstInfo(AbstractAdditionalTokensInfo.java:314)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalInfoWithBuild.restoreInfoForModuleManager(AbstractAdditionalInfoWithBuild.java:264)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.recreateAllInfo(AdditionalSystemInterpreterInfo.java:179)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.getAdditionalSystemInfo(AdditionalSystemInterpreterInfo.java:155)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.getAdditionalSystemInfo(AdditionalSystemInterpreterInfo.java:124)
	at com.python.pydev.analysis.additionalinfo.AdditionalSystemInterpreterInfo.recreateAllInfo(AdditionalSystemInterpreterInfo.java:171)
	at com.python.pydev.analysis.additionalinfo.builders.InterpreterObserver.notifyDefaultPythonpathRestored(InterpreterObserver.java:36)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.restorePythopathForInterpreters(AbstractInterpreterManager.java:740)
	at org.python.pydev.ui.interpreters.AbstractInterpreterManager.setInfos(AbstractInterpreterManager.java:657)
	at org.python.pydev.ui.pythonpathconf.AbstractInterpreterPreferencesPage$5.run(AbstractInterpreterPreferencesPage.java:282)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:119)

!ENTRY org.python.pydev.shared_core 4 4 2016-08-05 10:39:05.202
!MESSAGE Info: Rebuilding internal caches: D:\DEV\request\.metadata\.plugins\org.eclipse.core.resources\.projects\request\com.python.pydev.analysis\AdditionalProjectInterpreterInfo.pydevinfo (Expected error to be provided and got no error!)
